src/com/main/model/Set.java

```diff
@@ -0,0 +1,56 @@
+package com.main.model;
+
+public class Set {
+	
+	private int setId;
+	private String name;
+	private float price;
+	private int available;
+	private int vendorId;
+	public Set(int setId, String name, float price, int available, int vendorId) {
+		super();
+		this.setId = setId;
+		this.name = name;
+		this.price = price;
+		this.available = available;
+		this.vendorId = vendorId;
+	}
+	public int getSetId() {
+		return setId;
+	}
+	public void setSetId(int setId) {
+		this.setId = setId;
+	}
+	public Set(String name, float price, int available, int vendorId) {
```

Comment:
Can you move this under the first constructor to keep all the constructors together?